requirements.txt

```diff
@@ -16,7 +16,7 @@ gunicorn==19.7.1
 idna==2.6                 # via requests
 itsdangerous==0.24        # via flask
 jinja2==2.10              # via flask
-markupsafe==1.0           # via jinja2
+markupsafe==1.1.1           # via jinja2
```

Comment:
Ideally you should have updated requirements.in and generated requirements.txt using pip-compile.

Reply:
Adding another dependency is always my least Priority. It exposes our stack to the range of new vulnerabilities. This format is actually followed in all stacks.
But surely keep it in Potential TODOs. Thanks

Reply:
All our components follows the way I've mentioned. You shouldn't just update requirements.txt. @yzainee , please comment.

Reply:
Not Sure how pip-compile works, so can't just take risks of adding unknown dep. 
Btw pip-compile is no longer available https://pypi.org/project/pip-compile/

Reply:
yes. @deepak1725 please update the in file and run pip-compile. it will automatically generate the txt file

Reply:
Ack!